assets/src/styles/campaigns/themes/_theme_arctic.scss

```diff
@@ -73,16 +73,30 @@ body.theme-arctic {
   .page-header {
     @include page-header-block($save-the-arctic);
 
+    .container > *:not(.row) {
+      @include x-large-and-up {
+        max-width: 80%;
+      }
+    }
+
     .page-header-title {
-      font-size: 3.125rem;
-      line-height: 3.75rem;
       color: $arctic-blue;
+
+      @include mobile-only {
+        line-height: 1.38;
+        margin-bottom: 5px;
+        font-size: $font-size-xl;
```

Comment:
Maybe we can solve this too by introducing variables for the font size on each screen size? Then we can give it a default value in our base css, and only set this variable to something else for the arctic theme.

Reply:
As seen during sprint review, this will be done in a different PR